### PR TITLE
Added travis.yml and updated Dockerfile using Alpine.

### PR DESCRIPTION
**Pull request by zsuzhengdu**

### CODE REVIEW

**Comment by bkhouri** [comment]
Glad to see integratation with Travis CI 👍

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  DOCKER_COMPOSE_VERSION: 1.4.2
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+script:
+  - docker-compose run wildfly
```
Comment (bkhouri) [inline .travis.yml]: FYI, I think you can just do "docker-compose up" and it should start both wildfly and test containers.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+sudo: required
+services:
+  - docker
+
+env:
+  DOCKER_COMPOSE_VERSION: 1.4.2
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
```
Comment (bkhouri) [inline .travis.yml]: I understand you want a specific version o docker-compose, but are there any risks with replacing the existing version with 1.4.2?  Can there be incompatibility issues with the docker-compose binary you are downloading with the other docker-compose dependent files?

I don't think I communicated my thoughts clearly enough, so feel free to ping me if you want additional clarifications.

File: tests/unit/client_test.py [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+		responses.add(responses.POST, client.endpoint, json={}, status=200)
+		resp = client._post(json.dumps({}))
+
+		assert resp.status_code == 200
+		assert resp.json() == {}
+		assert responses.calls[0].request.url == client.endpoint
+		assert responses.calls[0].response.text == '{}'
+
+	@mock.patch('requests.Response.raise_for_status')
```
Comment (zsuzhengdu) [inline tests/unit/client_test.py]: @iangkent

**Pull request merged**